entities/entities/darkrp_laws/cl_init.lua

```diff
@@ -33,26 +33,21 @@ end
 local function addLaw(inLaw)
     local law = DarkRP.textWrap(inLaw, "TargetID", 522)
 
-    Laws[#Laws + 1] = law
+    local lawNumber = table.insert( Laws, law )
+    hook.Run("addLaw", lawNumber, law)
```

Comment:
This changes behaviour. `law` is text wrapped, `inLaw` is not. Please put it back to `inLaw`